docs/stacks-guide.md

```diff
@@ -199,11 +199,26 @@ If the control cluster doesn't recognize the Wordpress instance type, it
 could be because the stack is still being installed. Wait a few seconds,
 and try creating the Wordpress instance again.
 
-### Wait
+### Wait for Claims
 
 The Wordpress can take a while to spin up, because behind the scenes
-Crossplane is creating all of its dependendencies, which is a database
-and Kubernetes cluster. To check the status, you can use [the trace 
+the Crossplane Stack is creating all of its dependendencies using claim resources.
+
+The claims we are waiting for are:
+
+* `MySQLInstance` - a MySQL database, that could be provisioned by CloudSQL, RDS, or Azure Database, for example
+* `KubernetesCluster` - a claim for a Kubernetes cluster that could be provided by GKE, AKS, EKS, or others
+* `KubernetesApplication` - a claim for a Kubernetes application workload to deploy on a Kubernetes cluster
```

Comment:
This is a bit of a nitpick, but do we usually refer to a `KubernetesApplication` as a claim type? I don't necessarily think it is wrong, and I like how you refer to it as a "powerful claim" below, but I just don't usually think of it as a claim type. This could totally just be my own mental model getting in the way here so please feel free to push back :)

Reply:
I think I'd prefer to avoid referring to KA as a claim - it feels like it risks overloading the term.

Reply:
Revised here..
https://github.com/crossplaneio/crossplane/compare/0f9555d026b17907b2ab4959c88333c03afbfbd0..c498c764f3b9c33f5fa2e3772e16984f5fd1e50d